composer.json

```
@@ -14,7 +14,8 @@
     "require": {
         "php": ">7.2.5",
         "giggsey/libphonenumber-for-php": "^8.0",
-        "symfony/framework-bundle": "^3.4|^4.3|^5.0"
+        "symfony/framework-bundle": "^3.4|^4.3|^5.0",
+        "symfony/intl": "^3.4|^4.3|^5.0"
```

```suggestion
        "symfony/intl": "^4.3|^5.0"
```
Wouldn't this be enough?

---

I think we can keep it like that for now as for the other dependencies and drop symfony 3.4 support in a future PR/Release

---

@OskarStark see https://github.com/odolbeau/phone-number-bundle/pull/83